src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -432,6 +432,20 @@ public override int Order
         }
 
         /// <inheritdoc />
+        public override string Placeholder
+        {
+            get
+            {
+                if (DisplayMetadata.Placeholder == null)
+                {
+                    return null;
+                }
+
+                return DisplayMetadata.Placeholder();
```

Comment:
When I was messing around with my own micro ORM I found that `Invoke()` was slower than a simple `()` invocation; I don't know if that's changed at all so I'll leave it 'expanded' but I will add the comment. Thanks!